airbyte-integrations/connectors/source-monday/metadata.yaml

```diff
@@ -10,7 +10,15 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 80a54ea2-9959-4040-aac1-eee42423ec9b
-  dockerImageTag: 1.1.4
+  dockerImageTag: 2.0.0
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: "Connector migrated to the latest API Version 2024-01. Type of Id field changed from integer to string. After upgrading, please reset the stream."
```

Comment:
Source Monday has deprecated API version 2023-07.  We have upgraded the connector to the latest API version 2024-01.  In this new version, the Id field has changed from an integer to a string in the streams Boards, Items, Teams, Users, and Workspaces. Please reset affected streams.